Userland/Applications/Spreadsheet/main.cpp

```diff
@@ -22,18 +22,13 @@
 
 int main(int argc, char* argv[])
 {
-    if (pledge("stdio recvfd sendfd rpath unix cpath wpath thread", nullptr) < 0) {
+    if (pledge("stdio recvfd sendfd rpath fattr unix cpath wpath thread", nullptr) < 0) {
```

Comment:
Really nit-picking here, but could the pledges be better organized? For example put the read/create/write-path pledges together, so that it is easier "on the eyes".

Reply:
If we're gonna do that, we should reorder all the pledge lists in all the apps and utils, changing this one thing won't really help :P